### ISSUE TITLE
Document Amplify Auth for Login with Instagram

### ISSUE THREAD

**Issue by nihp** [aws-amplify/amplify-js#6393]
I am planning to add login features with social logins

For e.g) INSTAGRAM.

Can i able to authenticate the app using instagram in aws cognito.

**Comment by evcodes** [comment]
Hello nihp,

As per the documentation on this page: https://docs.amplify.aws/lib/auth/social/q/platform/js you should be able to add authentication via amplify add auth with Federate using social OAuth2.0. 

I am currently working on recreating this and will get back to you 👍

**Comment by nihp** [comment]
@evcodes Thanks for your reply.

So are you making the changes in the documentation for the instagram login steps? If so it would be great helpful for us.

Here I found facebook, google, apple, amazon
![Screenshot 2020-07-27 at 5 11 27 PM](https://user-images.githubusercontent.com/36992338/88538316-c4f8ce80-d02c-11ea-9adf-da5a9c4990de.png)

**Comment by nihp** [comment]
@evcodes 

But I found some related threads in SO refer [here1](https://stackoverflow.com/questions/59536786/aws-login-using-username-password-or-social-login
) [here2](https://stackoverflow.com/questions/46934013/how-to-make-instagram-login-with-aws-cognito-federated-identities)

```
unfortunately No, you need to create your custom auth provider. 

Unfortunately there is no out of box support for Instagram, so your best bet is to setup Instagram as Custom provider. Please refer to this doc and blog to learn about setting up custom provider.

Basically your app needs to interact directly with Instagram to get Instagram User's unique id. That id can then be used as developer user identifier in GetOpenIdTokenForDeveloperIdentity to get Cognito Identity Id and Token.
```
https://stackoverflow.com/questions/46934013/how-to-make-instagram-login-with-aws-cognito-federated-identities

@mauerbac Kindly re-open this ticket. There is no updates regarding INSTAGRAM login in any docs. Need to give an exact solution for this.

**Comment by mauerbac** [comment]
Updated @nihp

**Comment by Sumanth-Shetty** [comment]
hey guys, 
Was anyone able to achieve an Instagram federated login for Cognito?

**Comment by mateomorrison** [comment]
Still nothing here? Would love to use Instagram + Amplify

**Comment by te-nitin** [comment]
The instagram has stopped supporting authentication. You can use Instagram Basic Display API allows users of your app to get basic profile information, photos, and videos in their Instagram accounts.

Check below links -

1)https://developers.facebook.com/docs/instagram-basic-display-api 
2)https://www.instagram.com/developer/

They mentioned, "Authentication — Instagram Basic Display is not an authentication solution. Data returned by the API cannot be used to authenticate your app users or log them into your app. If you need an authentication solution we recommend using Facebook Login instead".

**Comment by imsziya** [comment]
Even I am looking to integrate Instagram as an Identity Provider in AWS Cognito but I am able to do.
Can I create a Custome Provider in Cognito Pool?
